python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -104,6 +104,24 @@ def test_serialize():
     )
 
 
+def test_empty_video_and_signal_str():
```

Comment:
blocking: these are two different cases, can you split them into two tests?

Reply:
Have split into two